camelot/core/serializable.py

```diff
@@ -49,10 +49,13 @@ def _to_dict(self):
 class DataclassEncoder(json.JSONEncoder):
 
     def default(self, obj):
+        from camelot.admin.action.list_filter import All
         if isinstance(obj, ugettext_lazy):
             return str(obj)
         if isinstance(obj, Enum):
             return obj.value
+        if obj == All:
```

Comment:
dit gaat niet goed komen heh, want dat kan nooit betrouwbaar gedeserialiseerd worden,
ik zou voorlopig eerder opteren om de 'All' class weg te gooien, en in de filters waar die
specifiek nodig is, een mode met name '__all' er tussen te zetten, zoals de '__clear'
in de SetFilters action

Reply:
Kunnen we dat All object dan niet vervangen door: `All = '__all'`?

Reply:
dat is idd mss goed enough for now, maar zet er dan mss wel een woordje van commentaar bij

Reply:
Ter verduidelijking: Ik zou die `All = '__all'` in list_filter.py zetten i.p.v:
```
class All(object):
    pass
```
Dan worden er modes aangemaakt met name '__all'.

Niet enkel bij het serializeren dus.